core/store/migrate/migrations/0275_add_workflow_specs_v2.sql

```diff
@@ -0,0 +1,22 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE workflow_specs_v2 (
```

Comment:
@cedric-cordenier I removed `secret_id`. I presume this wouldn't be used for secrets store. Correct me here if I'm wrong.